### ISSUE TITLE
Lua Repair is broken

### ISSUE THREAD

**Issue by Piengineer12** [Piengineer12/public-addons#94]
Latest update seems to have broken some stuff. Currently unknown if this is simply due to logging.

**Comment by Piengineer12** [comment]
```
[Lua Repair] lua/autorun/+piengineers_lua_repair.lua:120: attempt to index local 'NUMBER' (a nil value)
1. FixAllErrors - lua/autorun/+piengineers_lua_repair.lua:120
2. unknown - lua/autorun/+piengineers_lua_repair.lua:375
```

**Comment by Piengineer12** [comment]
Resolved via commit 809181b91a17f7e883c3209ac3348e5c9d7fbf4c.
